### PR TITLE
Account / User Management

### LINKED ISSUE #30: Manage Account Page
Personal Information (Editable)
- [x] First Name & Last Name
- [x] Phone # 
- [x] Languages

Assigned Classes (Only editable by admins)
- [x] List of classes
- [x] View button

### LINKED ISSUE #46: User Management
Allow only the **admins** to access a `User Management` page to:

- [x] Change user roles
- [ ] Superuser delete user

### LINKED ISSUE #95: Hovering over Username
When you mouse over the username in the top right, it grays out indicating that you can interact with it. When you click on the username, nothing happens. If you move over to the arrow, though, that's when the options come up.

### PR DESCRIPTION
**Pull request by michelleburd**
- Account Management
- User Management
- Hovering over username

Fixes #30, Fixes #46, Fixes #95

### CODE REVIEW

File: views/account/account_archive.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        $archiveRes = $db->query("UPDATE employees SET df = 1 WHERE employeeid = $1", [$employeeid]);
+        $result = $db->query("SELECT facilitatorid FROM facilitators WHERE facilitatorid = $1", [$employeeid]);
+        $isFacilitator = pg_fetch_assoc($result);
+
+        # Check if the employee is also a facilitator so we can remove them from that table as well
+        if ($isFacilitator) {
+            $facilitatorRes = $db->query("UPDATE facilitators SET df = 1 WHERE facilitatorid = $1", [$employeeid]);
+        }
+
+        if ($archiveRes && (($isFacilitator && $facilitatorRes) || !$isFacilitator)) {
```
Comment (ghost) [inline views/account/account_archive.php]: `$success = $archiveRes && (($isFacilitator && $facilitatorRes) || !$isFacilitator)`

File: views/account/account_modify.php [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+if (($_SESSION['employeeid'] != $employeeid) && (!(hasRole(Role::Admin)))) {
+    header('Location: /dashboard');
+    die();
+} else {
+    $loggedInUserRole = $_SESSION['role'];
+
+    # Get employee/people information
+    $db->prepare("get-employee-info", "SELECT people.firstname, people.middleinit, people.lastname, employees.email, employees.primaryphone, employees.permissionlevel " .
+        "FROM employees " .
+        "LEFT JOIN people ON employees.employeeid = people.peopleid
```
Comment (ghost) [inline views/account/account_modify.php]: I think this line should end `people.peopleid" . ` to follow what seems standard for multi-line strings

**Comment by stickzman** [comment]
If I click view > edit > back, I get stuck in a loop where pressing back on the edit page "account-settings/modify/1" brings me to the view page "account-settings/1", and back on that page brings back to the edit page again.

File: views/user/user_restore.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            <div class="card-footer text-right">
+                <button type="button" class="btn btn-light" onclick="goBack()">Cancel</button>
+                <button type="submit" name="restore" class="btn cpca">Restore</button>
+            </div>
+        </form>
+    </div>
+
+<?php
+include('footer.php');
+include ('footer.php');
```
Comment (stickzman) [inline views/user/user_restore.php]: Delete the duplicate line

File: views/account/account_settings.php [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                            </div>
+                        </div>
+                    </div>
+                    <br>
+                </div>
+            </div>
+        </div>
+        <script>
+            function deleteModal() {
+
```
Comment (ghost) [inline views/account/account_settings.php]: Empty function here

**Review by blackjack26 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by stickzman — APPROVED** [review]

**Pull request merged**